### ISSUE TITLE
Want to get detail help

### ISSUE THREAD

**Issue by VonLYC** [angelopassaro/SEC-UAV#1]
I am very interested in this project you have done. Can you give me your contact information to give me some guidance?

**Comment by Phanx00** [comment]
Hi, I'm glad you're interested in the work I've done. Can you explain what problem you are experiencing?

**Comment by VonLYC** [comment]
I have just started to get involved in drone simulation, and I am currently very interested in the direction of Mavlink encryption, but I don’t understand some of the source code of your project directly, that is, the entire drone communication system (I mean no one The direct interaction mode between the machine and the ground station) I don’t know much about it. Where should I start to learn to understand the source code of the project?

**Comment by VonLYC** [comment]
My question does not refer to where there is a problem (errors) in your code, but I want to learn how to thoroughly understand your source code from a person who only has a basic knowledge of C++ and cryptography   Sincerely

**Comment by Phanx00** [comment]
I recommend you to study C well, in fact it helped me a lot for the development of the project, while C++ was of little use for the modifications made. As far as cryptography is concerned I studied the following book "Introduction to modern cryptography" by Jonathan Katz, during my course of study which helped me a lot. Subsequently for integrated encryption algorithms I studied the relevant papers. I hope I have helped you a little. Sincerely
